docs/developers/node-tutorial.md

```diff
@@ -0,0 +1,389 @@
+# Getting and Sending Transactions with Celestia Node
+
+In this tutorial, we will cover how to use the Celestia Node API to submit and
+retrieve messages from the Data Availability Layer by their namespace ID.
+
+## Hardware Requirements
+
+You can find hardware requirements [here](https://docs.celestia.org/nodes/bridge-validator-node#install-golang).
+
+## Setting Up Dependencies
+
+You can follow the tutorial for setting up the dependencies [here](https://docs.celestia.org/nodes/bridge-validator-node#setting-up-your-bridge-node).
+
+## Celestia Node
+
+### Install Celestia Node
+
+We need to install Celestia Node.
+Run the following command:
+
+```sh
+cd $HOME
+git clone https://github.com/celestiaorg/celestia-node.git
+cd celestia-node/
+make install
+```
+
+Verify its installed by running the following command:
+
+```console
+$ celestia version
+Semantic version: v0.2.0
+Commit: 1fcf0c0bb5d5a4e18b51cf12440ce86a84cf7a72
+Build Date: Fri 04 Mar 2022 01:15:07 AM CET
+System version: amd64/linux
+Golang version: go1.17.5
+```
+
+### Instantiate Celestia Light Node
+
+Now, let's instantiate a celestia-light node:
+
+```sh
+./build/celestia light init
+```
+
+### Generate A Wallet
+
+We need to generate a wallet.
+Run the following command:
+
+```sh
+celestia light keys add developer --keyring-backend test
+```
+
+This generates a key called `developer` in `~/.celestia-light/keys/keyring-test`.
+
+Now, head over to the Celestia Discord channel `#faucet`.
+
+You can request funds to your wallet address using the following command in Discord:
+
+```console
+$request <Wallet-Address>
+```
+
+Where `<Wallet-Address>` is the `celes1******` address generated
+when you created the wallet.
+
+With your wallet funded, you can move on to the next step.
+
+### Connect To A Public Core Endpoint
+
+Let's now run the Celestia Light node with a GRPC connection to
+an example public Core Endpoint.
+
+> Note: You are also encouraged to find a community-run API endpoint
+  and there are several in the Discord. This one is used for demonstration
+  purposes.
+
+Here we are starting a light node with a connection to a Core endpoint at
+64.227.74.87:9090 and also telling the light node to use the `developer`
+key we generated as its default account.
+
+```sh
+./build/celestia light start --core.grpc 64.227.74.87:9090 --keyring.accname developer
+```
+
+## Node API Calls
+
+Open up another terminal window in order to begin querying the API.
+`celestia-node` exposes its RPC endpoint on port `26658` by default.
+
+### Balance
+
+Now, let's query our node for the balance of its default account
+(which is the account associated with the `developer` key we generated earlier):
+
+```sh
+curl -X GET http://127.0.0.1:26658/balance/
```

Comment:
@renaynay don't we need to include the celes address here as in `/balance/<ADDRESS>`